javagen/src/main/resources/readme.md

```diff
@@ -0,0 +1,96 @@
+# Azure ${service-name} client library for Java
+
+${readme-description}
+
+**Please rely heavily on the [service's documentation][product_documentation] and [Low-Level client documentation][low_level_client] to use this library**
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_reference_doc] | [Product Documentation][product_documentation] | [Samples][samples_readme]
+
+## Getting started
+
+### Prerequisites
+
+- A [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Azure Subscription][azure_subscription]
+
+### Include the Package
+
+[//]: # ({x-version-update-start;${group-id}:${artifact-id};current})
+```xml
+<dependency>
+  <groupId>${group-id}</groupId>
+  <artifactId>${artifact-id}</artifactId>
+  <version>${version}</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+
+### Authenticate the client
+In order to interact with this service, your client must present an Azure Active Directory bearer token to the service.
+
+The simplest way of providing a bearer token is to use the `DefaultAzureCredential` authentication method by providing client secret credentials is being used in this getting started section but you can find more ways to authenticate with [azure-identity][azure_identity].
+
+#### Create GlossaryBaseClient with Azure Active Directory Credential
+
+You can authenticate with Azure Active Directory using the [Azure Identity library][azure_identity].
+
+To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below, or other credential providers provided with the Azure SDK, please include the `azure-identity` package:
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;dependency})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>${azure-identity-version}</version>
+</dependency>
+```
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
+
+##### Example
+<!-- embedme ./src/samples/java/${namespace-path}/ReadmeSamples.java#L${line-start}-L${line-end} -->
+```java
+${example-code}
+```
```

Comment:
I don't think it is a valid assumption that SDK can only contain 1 sample.

Reply:
Updated